static/components.js

```diff
@@ -175,6 +175,23 @@ module.exports = {
             }
         };
     },
+    getSelectionDAGView: function () {
```

Comment:
Thanks: this file was never meant to be so big and sprawling...I appreciate your patience in navigating it...